docs/nodes/light-node.md

```diff
@@ -17,7 +17,7 @@ to interact with the Celestia network.
 
 Light Nodes have the following properties:
 
-1. Connect to a [Celestia Bridge Node](https://github.com/celestiaorg/networks#celestia-validator-bridge-nodes)
+1. Connect to a [Celestia Bridge Node](https://github.com/celestiaorg/networks#celestia-validator-light-nodes)
```

Comment:
This is no longer true btw, they only need to connect to a `--core.grpc` endpoint for state interaction now, so u can remove this point.